controllers/suite_test.go

```diff
@@ -65,7 +65,9 @@ func TestMain(m *testing.M) {
 		filepath.Join("testdata", "crds"),
 	))
 
-	managed.InitManagedTransport()
+	if err := transport.InitManagedTransport(); err != nil {
+		panic(fmt.Sprintf("Failed to initialize libgit2 managed transport: %v", err))
```

Comment:
```suggestion
		panic(fmt.Sprintf("failed to initialize libgit2 managed transport: %v", err))
```

Reply:
I kept it as "Failed" to match the panic statements below